### ISSUE TITLE
[Docs] Provide some examples how to run in production

### ISSUE THREAD

**Issue by Koc** [dunglas/symfony-docker#19]
Please describe in documentation sollution for the next cases:
1. running some workers in parallel, for example for consuming messages by Messenger component from 4.1
2. running multiple php application backends, for example api
3. running periodical tasks by cron

**Comment by aCherfiScala** [comment]
I think that will be very useful if you explain how to deploy containers in production environment using docker orchestrator like Kubernetes or Swarm.

**Comment by younes-dev** [comment]
please i have problem to connect my project symfony to mysql container using docker could anyone help  ?
 
root@9b0782a498c3:/application#  **php bin/console doctrine:database:create**   
[error] Error thrown while running command "doctrine:database:create". Message: "An exception occurred in driver: SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known"


In AbstractMySQLDriver.php line 93:
                                                                                                                                   
  An exception occurred in driver: SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known  
                                                                                                                                   

In PDOConnection.php line 31:
                                                                                                  
  SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known  
                                                                                                  

In PDOConnection.php line 27:
                                                                                                  
  SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known  
                                                                                                  

In PDOConnection.php line 27:
                                                                                               
  PDO::__construct(): php_network_getaddresses: getaddrinfo failed: Name or service not known  
                                                                                               

doctrine:database:create [-s|--shard SHARD] [-c|--connection [CONNECTION]] [--if-not-exists] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-e|--env ENV] [--no-debug] [--] <command>

**Comment by maxhelias** [comment]
@younes-dev I think this is another issue that is unrelated to this one. Please open an other issue and describe how you connect your database.
Thanks

**Comment by younes-dev** [comment]
Thank you @maxhelias  This my issue :
[Symfony 4 connection on Mysql docker container](https://github.com/younes-dev/mystore/issues/1https://github.com/younes-dev/mystore/issues/1)

I'm stack in this Error any help please ?

**Comment by silverbackdan** [comment]
@younes-dev you must raise a separate issue in this repository, not your own. Your own repo is private and cannot be seen by anyone else.

**Comment by maxhelias** [comment]
see #77

**Comment by arnaudbagnis** [comment]
> I think that will be very useful if you explain how to deploy containers in production environment using docker orchestrator like Kubernetes or Swarm.

I agree with you, showing a **helm charts template** for k8s would be perfect 👍
